blogposts/2021/release-3.29.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Sourcegraph 3.29 release"
+publishDate: 2021-06-18T10:00-07:00
+description: "The Sourcegraph 3.29 release introduces improved search results ranking, makes it easier to create Code Insights, and includes support for bulk actions with Batch Changes."
+tags: [blog, release]
+slug: "release/3.29"
+published: false
+changelogItems:
+  - description: "Code Insights queries can now run concurrently up to a limit set by the `insights.query.worker.concurrency` site config."
+    url: https://github.com/sourcegraph/sourcegraph/pull/21219
+    category: Code Insights
+  - description: "Code Insights workers now support a rate limit for query execution and historical data frame analysis using the `insights.query.worker.rateLimit` and `insights.historical.worker.rateLimit` site configurations."
```

Comment:
@abschu I'll leave the gameday decision to you but I would suggest keeping it because:
a) it's a setting that an end user / admin can set, so it's not just something happening behind the scenes
b) my guess is primarily admins (people who could make this setting update if they wanted) are reading the changelog closely so they know why/when to upgrade.

Reply:
Let's keep - thank you!